.github/workflows/CompatHelper.yml

```
@@ -1,22 +1,25 @@
 name: CompatHelper
-
 on:
   schedule:
-    - cron: '0 0 * * *'
-  push:
-    branches:
-      - actions/trigger/CompatHelper
-
+    - cron: 0 0 * * *
+  workflow_dispatch:
 jobs:
   CompatHelper:
     runs-on: ubuntu-latest
     steps:
-      - uses: julia-actions/setup-julia@latest
-        with:
-          version: 1.4
-      - name: Pkg.add("CompatHelper")
-        run: julia -e 'using Pkg; Pkg.add("CompatHelper")'
-      - name: CompatHelper.main()
+      - name: "Install CompatHelper"
+        run: |
+          import Pkg
+          name = "CompatHelper"
+          uuid = "aa819f21-2bde-4658-8897-bab36330d9b7"
+          version = "2"
+          Pkg.add(; name, uuid, version)
+        shell: julia --color=yes {0}
+      - name: "Run CompatHelper"
+        run: |
+          import CompatHelper
+          CompatHelper.main()
+        shell: julia --color=yes {0}
         env:
-          GITHUB_TOKEN: ${{ secrets.CUSTOM_GITHUB_TOKEN }}
-        run: julia -e 'using CompatHelper; CompatHelper.main()'
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          COMPATHELPER_PRIV: ${{ secrets.DOCUMENTER_KEY }}
```

From https://github.com/tkf/BenchmarkCI.jl/blob/d8df5445791a6a20d243612da640b2c92dd69e83/.github/workflows/docs.yml#L23-L27

```suggestion
          COMPATHELPER_PRIV: ${{ secrets.SSH_KEY }}
```